accord-core/src/main/java/accord/impl/SimpleProgressLog.java

```diff
@@ -812,18 +812,26 @@ public void onCallbackFailure(Id from, Throwable failure)
 
     public static class ApplyAndCheck extends Apply
     {
-        final Set<Id> notPersisted;
+        public final Set<Id> notPersisted;
```

Comment:
Reply from Blake in JIRA

The integration of the progress log hasn't started yet. I'd expect the interface and reference implementation to be refined as that happens, but the objective for this patch to do the bare minimum to prevent build issues and test failures.